apis/model/v1alpha1/modelversion_types.go

```diff
@@ -23,8 +23,13 @@ import (
 const (
 	// KubeDLModelPath is the env key to indicate model path in container.
 	KubeDLModelPath = "KUBEDL_MODEL_PATH"
-	// DefaultModelInImagePath is the default model artifact path when build image.
-	DefaultModelInImagePath = "/model"
+
+	// DefaultModelPathInImage is the default dir inside the training container for storing the model artifacts.
+	// This is a mount path inside the container, mounted from an external storage such as local fs, nfs.
+	// The training code is expected to store the model under this directory, such as setting the model_export_path of 'tensorflow saved_model' to this value.
+	// The ModelVersion Controller will then create the image based on the corresponding external storage path defined in .ModelVersionSpec.Storage.
+	// TODO: support user-specified path in container.
+	DefaultModelPathInImage = "/kubedl-model"
```

Comment:
should model path `DefaultModelPathInImage` be a directory ("/kubedl/model") , and run `cp -R {path}` in `serving/model.go` to copy multiple models recursively .